Libraries/CameraRoll/CameraRoll.js

```diff
@@ -12,7 +12,7 @@
  */
 'use strict';
 
-const {PropTypes, checkPropTypes} = require('React');
+const PropTypes = require('prop-types');
```

Comment:
`PropTypes` is also used in the file.

This would be possible:
```js
const PropTypes = require('React');
const {checkPropTypes} = PropTypes;
```